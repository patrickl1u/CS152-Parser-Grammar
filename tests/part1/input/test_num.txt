99
102
3
10294
