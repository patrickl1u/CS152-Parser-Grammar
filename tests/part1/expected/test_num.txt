Ctrl-D to quit. 
NUMBER 99
NUMBER 102
NUMBER 3
NUMBER 10294
quitting...
